File: XBOOK/XBOOK.Web/ClientApp/src/app/modules/_shared/models/client/client-view.model.ts
import { BaseModel } from '@shared/model/base.model';
export class ClientView extends BaseModel {
  clientName: string;
  address: string;
  taxCode: number;
  tag: number;
  contactName: string;
  email: string;
  note: string;
  clientId: number;
}
export class ClientViewModel {
  clientID: number;
  clientName: string;
  address: string;
  note: string;
  outstanding: number;
  overdue: number;
}

File: XBOOK/XBOOK.Dapper/ViewModels/ClientViewModel.cs
using System;
using System.Collections.Generic;
using System.Text;

namespace XBOOK.Dapper.ViewModels
{
    public class ClientViewModel
    {
        public int ClientID { get; set; }
        public string ClientName { get; set; }
        public string Address { get; set; }
        public string Note { get; set; }
        public decimal Outstanding { get; set; }
        public decimal Overdue { get; set; }
        public string TaxCode { get; set; }
        public string Email { get; set; }
        public string Tag { get; set; }
        public string ContactName { get; set; }
        public string BankAccount { get; set; }
    }
}
